cloud/logging.rst

```diff
@@ -83,16 +83,16 @@ Replace :code:`<token>` with the custom token listed in the provider's web site.
 | Message Template | ``<${PRI}>1 ${ISODATE} ${HOST} ${PROGRAM} ${PID} ${MSGID} [<token>@41123] $MSG\\n`` |
 +------------------+-------------------------------------------------------------------------------------+
 
-**Logentries**
+**Logentries** / **Rapid7 insightOps**
 
 +------------------+------------------------------------+
-| Hostname         | data.logentries.com                |
+| Hostname         | us.data.logs.insight.rapid7.com    |
```

Comment:
Was Logentries acquired or something? Why is their hostname now rapid7?

Reply:
Yeah, I do think so.